src_new/controllers/gpt.js

```diff
@@ -124,6 +124,10 @@ function getAdSlotSizesArray(divID, currentGoogleSlot) { // TDD, i/o : done
 
     if (sizeMapping !== false) {
         util.log(divID + ": responsiveSizeMapping applied: ");
+        /* istanbul ignore else */
+        if(typeof(sizeMapping[0]) == "string" || (sizeMapping[0].length == 1 && typeof(sizeMapping[0][0]) == "string")){
```

Comment:
Please add unit test cases for this scenario

Reply:
Done